src/functionsDelete.ts

```diff
@@ -0,0 +1,41 @@
+import * as helper from "./functionsDeployHelper";
+import { Queue } from "./throttler/queue";
```

Comment:
Not sure, but i agree that this isn't a very clear name. It will have a larger impact radius than i want for this PR since this structure is used across a bunch of code paths, but I think we ought to to rename Queue to ThrottlerQueue and Stack to ThrottlerStack once the functions refactor is fully merged in.